test/scripts/openshift-ci/deploy.serverless.sh

```diff
@@ -98,6 +98,12 @@ spec:
       enabled: true
 EOF
 
+# Apparently, as part of KNative installation, deployments can be restarted because
+# of configuration changes, leading to waitpodready to fail sometimes.
+# Let's sleep 2minutes to let the KNative operator to stabilize the installation before
+# checking for the readiness of KNative stack.
+sleep 120
```

Comment:
why don't use a more accurate way to check if the pod is ready?
something like `oc wait` or [this kind of script](https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/scripts/utils.sh#L60)
This is [an example](https://github.com/opendatahub-io/caikit-tgis-serving/blob/main/demo/kserve/scripts/test/grpc-call.sh#L13-L14)